### DISCUSSION TITLE
Setting up on Mac

### DISCUSSION THREAD

**Discussion by ghost** [opensafely/documentation#214] [Installing OpenSAFELY] [answered]
Hi. I've installed anaconda command line and can run "opensafely" ok (e.g. opensafely --help worked).

I installed the Mac Docker app ok. It seemingly runs ok and sits there saying "No containers running".

The opensafely "Getting Started" doc (for Windows) under "Install Docker" step 4. says "Test Docker and opensafely work together". So, in a Mac Terminal (not Anaconda Prompt, as I'm not sure I've got that on Mac?) I type "opensafely pull" and the message:

"No OpenSAFELY docker images found to update."

I feel that this is wrong and something should happen at this point...

best wishes, Guy Nason

**Comment by bloodearnest** [comment, marked as answer]
Hi Guy

The docs to test docker is install are incorrect.

To test docker is working you should instead try this:

    docker run hello-world

If you see a "Hello from Docker!" message, then docker is working on your system.

I shall update the docs shortly to use this command.

For the record, once you have run some OpenSAFELY studies, the command `opensafely pull` will update any OpenSAFELY images your have used to their latest version. But as you have not run any, it correctly reported that you don't yet have any OpenSAFELY images to update.

Thanks for the report.

**Reply by ghost** [reply]
Thank you ever so much. I have checked and Docker seems to be working properly (I also tried the ubuntu example, which was rather cool). Thank you very much indeed. Guy Nason
